app/views/console/_console.html.erb

```diff
@@ -6,16 +6,16 @@
                     style: 'display:none',
                     data: {
                       url: status_console_index_path,
-                      close_url: close_url,
+                      close_url: local_assigns.fetch(:close_url, ''),
```

Comment:
this seems unnecessary elsewhere, looks like we're missing something in the caller to this partial.

Reply:
if it's because we have a single partial handling all the consoles, that's great, but then someone else (either the server via RJS or the client JS should tie each instance where the console can appear with a :close_url data attribute somehow?